umh.docs.umh.app/content/en/docs/getstarted/installation.md

```diff
@@ -26,15 +26,19 @@ installing your first instance and connecting to an OPC UA simulator in no time.
 
 - ARM-based systems, such as a Raspberry Pi, are not currently supported.
 
+- Ensure the root partition (`/`) has at least <span id="storage-requirements"></span> of free space. The default Rocky Linux installation creates a large `/home` partition, so custom partitioning during the installation of Rocky Linux may be required.
+- For network connectivity with other machines, use a bridged network adapter in the virtual machine. NAT or host-only adapters will prevent external connections to your instance.
```

Comment:
This here is only relevant for VMs and I think specifically VirtualBox, but there are also other VMs and installation methods (we have some guides on learn).

Maybe move this somewhere else? Otherwise we have a very specific thing for a subset in the overall installation tutorial